src/public/lib/SpellCheck.js

```diff
@@ -70,7 +86,7 @@ class SpellCheck {
      * dictionary that is valid based on system (defaults to en_US)
      */
     setEnabledDictionaries () {
-        const dictionaries = this.dictionaries;
+        const dictionaries = this.availableDictionaries;
```

Comment:
When testing this it seems to have enabled all dictionaries for me. I think it is caused by the change you made here.